server/src/main/scala/net/wiringbits/repositories/daos/DatabaseTablesDAO.scala

```diff
@@ -37,7 +44,7 @@ object DatabaseTablesDAO {
       TableMetadata(tableName, columnsMetadata.toList, tableData.toList)
     } finally {
       resultSet.close()
-      statement.close()
+      preparedStatement.close()
```

Comment:
This shouldn't invoke `val tableData = getTableData(resultSet)`, instead, the repository should get the metadata, then, use that to invoke `getTableData`